routes/api/util/form-validation/edit.js

```diff
@@ -0,0 +1,27 @@
+const Validator = require('validator');
+const isEmpty = require('is-empty');
+
+/**
+ * Takes in edit form input and checks for validity
+ * @param requestBody the data object holding all form inputs as fields
+ */
+function validateEditInput(requestBody) {
+  const errors = {};
+  // Only validate email and gradYear because they have more strict input
+  if (!Validator.isEmail(requestBody.email)) {
```

Comment:
Also for all of these we'd have to check if they're not empty before considering validating them like
```
if (!isEmpty(requestBody.email) && !Validator.isEmail(requestBody.email))
```